node/test/send.js

```diff
@@ -203,57 +206,64 @@ allocCluster.test('request().send() to a pool of servers', 4, function t(cluster
             res.sendOk(arg2, arg3 + ' served by ' + chanNum);
         });
         channel.peers.add(chan.hostPort);
+        var peer = channel.peers.get(chan.hostPort);
+        var conn = peer.connect(chan.hostPort);
+        console.log(chan.hostPort);
+        conn.on('identified', ready.signal);
```

Comment:
this whole pattern might be worthy of getting hauled back into allocCluster or an ancillary utility function around it.